app/openlxp_xds_project/settings.py

```diff
@@ -182,3 +183,27 @@
 
 EMAIL_BACKEND = 'django_ses.SESBackend'
 
+SAML2_AUTH = {
```

Comment:
Just something to consider here -- it will definitely be the case that the system will need to support multiple IdP configurations, each with their own metadata URL, X.509 certificate data, and attribute mappings.  So we'll want to consider how best to support that (such as with a Django model)

Reply:
Sounds good, there's probably some more research that has to be done there so I created a story here -https://jira.deloitte.com/browse/ECC-907